ccd-definition/CaseEvent/Camunda/CaseEvents.json

```diff
@@ -35,6 +35,20 @@
     "PreConditionState(s)": "*",
     "PostConditionState": "PROCEEDS_IN_HERITAGE_SYSTEM",
     "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
+    "ShowSummary": "N",
+    "ShowEventNotes": "N",
+    "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "FULL_DEFENCE_RESPONSE",
```

Comment:
what is this event for? name doesn't make sense to me

Reply:
Place holder for full defence response to standardise the camunda flow